app/main/browser-window.js

```diff
@@ -52,8 +61,13 @@ export async function createBrowserWindow(userAgentClient, onload) {
     if (onload) {
       onload();
     }
+    // The client needs to know where the UA service is in order to connect
```

Comment:
Wanted to separate that out so we load the browser chrome first ASAP, incase we needed to wait for the address for the UA service. Probably a moot point if we don't intend on having that being too variable.